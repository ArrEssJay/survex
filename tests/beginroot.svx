; *prefix \ or *begin \ would cause 0.96 to dump core
; though oddly it works for me with 0.96 when run from the test script...
*prefix \
*begin \
*fix 1 0 0 0
*end \
