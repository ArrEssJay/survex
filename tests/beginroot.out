./beginroot.svx:3:8: warning: *prefix is deprecated - use *begin and *end instead
./beginroot.svx:3:9: warning: ROOT is deprecated
./beginroot.svx:4:8: warning: ROOT is deprecated
./beginroot.svx:6:6: warning: ROOT is deprecated

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...
./beginroot.svx:5: warning: Unused fixed point "1"

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 5 warning(s).
